tensorboard/components/tf_backend/runsStore.ts

```diff
@@ -26,6 +26,12 @@ namespace tf_backend {
       });
     }
 
+    _getExperimentId() {
+      return (
+        new URLSearchParams(window.location.search).get('experiment') || ''
```

Comment:
Could we expose this somewhere so that the scalars code can re-use it rather than replicating it?  It could potentially be here although experimentsStore would perhaps be the best fit, even though this isn't strictly speaking the same as the old experiment data being served by that API.

That way it's easier to refactor later if we migrate away from a query parameter to e.g. embedding the experiment ID in the path.

Reply:
Exposed `tf_backend.getExperimentId()`. It’s still a bit of a kludge,
but it’s one kludge instead of two. :-)